src/classes/RD2_DataMigration_TEST.cls

```diff
@@ -867,7 +867,83 @@ private class RD2_DataMigration_TEST {
         List<Error__c> errors = errorGateway.getRecords();
         System.assertEquals(1, errors.size(), 'Validation error should be logged');
 
-        System.assertEquals(true, errors[0].Full_Message__c.contains(System.Label.RD_DonorIsRequired), 'Error message should match');
+        assertDonorIsRequiredError(errors);
```

Comment:
Can lines 872-875 be extracted into `assertDonorIsRequiredError()` and removed anywhere else they are used? The method might also have lines 867 and 868.